packages/arb-token-bridge-ui/src/components/App/App.tsx

```diff
@@ -294,6 +294,8 @@ function ConnectionFallback(props: FallbackProps): JSX.Element {
             <NetworkSelectionContainer>
               <HeaderNetworkNotSupported />
             </NetworkSelectionContainer>
+
+            <HeaderAccountPopover isCorrectNetworkConnected={false} />
```

Comment:
Added the existing HeaderAccountPopover in wrong-network state with a very small variation.